vulnerabilities/importers/github.py

```diff
@@ -227,10 +233,18 @@ def process_response(resp: dict, package_type: str) -> Iterable[AdvisoryData]:
             else:
                 logger.error(f"Unknown identifier type {identifier_type!r} and value {value!r}")
 
+        weaknesses = []
```

Comment:
Please make a separate function `get_cwes_from_github_advisory` and add docstring and tests for that.

Reply:
Done